pandas/tests/io/excel/test_xlsxwriter.py

```diff
@@ -63,3 +67,29 @@ def test_write_append_mode_raises(ext):
     with ensure_clean(ext) as f:
         with pytest.raises(ValueError, match=msg):
             ExcelWriter(f, engine='xlsxwriter', mode='a')
+
+
+def test_merged_cell_custom_objects(ext):
+    # Test that custom object types residing within merged (grouped)
```

Comment:
I think generally the comments here don't add much value. Can just replace this with `GH 27006` and remove comments below

Reply:
done